src/components/Toast/ToastRegion.tsx

```diff
@@ -0,0 +1,63 @@
+import { useRef } from "react";
+import type { AriaToastRegionProps } from "@react-aria/toast";
+import type { ToastState } from "@react-stately/toast";
+import { useToastRegion } from "@react-aria/toast";
+import { Toast } from "./Toast";
+import { classes } from "./toast.st.css";
+import Icon from "../Icon/Icon";
+
+interface ToastRegionProps<T> extends AriaToastRegionProps {
+  state: ToastState<T>;
+}
+
+function ToastRegion<T>({ state, ...props }: ToastRegionProps<T>) {
+  const ref = useRef(null);
+  const { regionProps } = useToastRegion(props, state, ref);
+
+  return (
+    <div {...regionProps} ref={ref} className={classes["toast-region"]}>
+      {state.visibleToasts.map((toast) => {
+        const { priority = 0 } = toast;
+        return (
+          <Toast
+            key={toast.key}
+            toast={toast}
+            state={state}
+            icon={
+              <>
+                {/* info */}
+                {priority === 0 && (
+                  <Icon>
+                    <path d="M8 1c3.9 0 7 3.1 7 7s-3.1 7-7 7-7-3.1-7-7 3.1-7 7-7zM8 0c-4.4 0-8 3.6-8 8s3.6 8 8 8 8-3.6 8-8-3.6-8-8-8v0z"></path>
```

Comment:
Nice job on sourcing these as defaults. :-)

Reply:
Could we componentise them and add them to icons and then make them configurable so that the likes of lace can override them as and when it needs to.

Reply:
Done.